### PR TITLE
feat: added pug

### PR DESCRIPTION
**Pull request by Aleksey28**
1. installed package pug
2. added template for pug
3. added using pug for server
4. added sending runs list to the client

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -30,15 +30,45 @@ class Server {
...
         const apiGithubObj = new ApiGithub(token);
-        const listWFRuns = await apiGithubObj.getListRuns(owner, repo);
-        const runsKey = 'workflow_runs';
+        const listWFRuns   = await apiGithubObj.getListRuns(owner, repo);
+        const runsKey      = 'workflow_runs';
 
         debugApp(`Runs count: ${listWFRuns[runsKey] && listWFRuns[runsKey].length}`);
 
-        this._app.use(KoaStatic(path.resolve() + '/public'));
+        //NOTE: rendering output page
```
Comment (AndreyBelym) [inline src/server.js]: It's good that you tried to provide some working example, but it's better to do not do this in your production code. Event if it is in a part of PR series that will be removed and refactored anyway. Just write a test that provides some dummy model (placeholder data) to your template, and assert the server's response.

**Review by AndreyBelym — COMMENTED** [review]
You're learning very fast, it's so amazing! So let's try to do not get used to bad habits from the starts. Try to never leave your prototypes in production and add them to the main branch of your VCS. It's just about keeping you public history clean and meaningful

**Comment by Aleksey28** [comment]
FPR

**Comment by Aleksey28** [comment]
FPR

**Review by AndreyBelym — APPROVED** [review]
Awesome 👍

**Pull request merged**